WalletWasabi.Fluent.Generators/Generators/AutoInterfaceGenerator.cs

```diff
@@ -140,4 +135,22 @@ public partial interface {{interfaceName}}
 
 		AddSource($"{className}.AutoInterface.g.cs", source);
 	}
+
+	private static bool IsInterfaceMethodImplementation(IMethodSymbol method)
+	{
+		foreach (INamedTypeSymbol implementedInterfaceSymbol in method.ContainingType.AllInterfaces)
```

Comment:
Could we replace nested loop via using linq expressions or collection extension methods.